src/components/messages/TextMessage.vue

```diff
@@ -111,6 +100,9 @@ _:-ms-fullscreen, :root .twc-message.twc-user .twc-text-message-wrapper  {
   line-height: 1.4;
   -webkit-font-smoothing: subpixel-antialiased;
   box-shadow: 0 2px 4px 0 rgba(85, 87, 85, 0.5);
+  overflow-wrap: break-word;    
+  overflow-wrap: break-all; 
```

Comment:
Does this work? Defining the same property twice with different values?

Reply:
Oversight, will remove

Reply:
removed https://github.com/artificialsolutions/teneo-web-chat/commit/b3ed2933ad3381b42154107e841cdac76a7bb194